exec/cnex/global.c

```diff
@@ -368,50 +367,18 @@ void array__toBytes__number(TExecutor *exec)
 
 void array__toString__number(TExecutor *exec)
 {
-    size_t x;
-    /* ToDo: Fix this to use proper string appends!! */
-    Cell *a = top(exec->stack);
-    char *s = malloc(STRING_BUFFER_SIZE);
-    if (s == NULL) {
-        fatal_error("Could not allocate %d byte string for cNumber array", STRING_BUFFER_SIZE);
-    }
-    strcpy(s, "[");
-    for (x = 0; x < a->array->size; x++) {
-        if (strlen(s) > 1) {
-            strcat(s, ", ");
-        }
-        strcat(s, number_to_string(a->array->data[x].number));
-    }
-    strcat(s, "]");
-    pop(exec->stack);
-    push(exec->stack, cell_fromCString(s));
-    free(s);
+//    Cell *a = cell_fromCell(top(exec->stack)); pop(exec->stack);
```

Comment:
Why is this code commented out? If it doesn't need to be here, it should be removed.